chalice/.chalice/policy.json

```diff
@@ -14,8 +14,8 @@
       "Effect": "Allow",
       "Action": "s3:*",
       "Resource": [
-        "arn:aws:s3:::hca-dss-test",
-        "arn:aws:s3:::hca-dss-test/*"
+        "arn:aws:s3:::$DSS_S3_TEST_BUCKET",
```

Comment:
It is written to AWS (by Chalice) only when deploying locally. When deploying on Travis CI, it skips that step (to avoid needing IAM write permissions).